shared/desktop/renderer/index.js

```diff
@@ -53,6 +53,42 @@ function setupAvatar() {
   initAvatarLoad(loadUserImageMap, loadTeamImageMap)
 }
 
+var timeoutID
+
+// Consider us inactive after a minute of no input
+// for the purpose of marking chat messages read
+function setTimeoutListeners() {
+  window.addEventListener('mousemove', resetTimer, false)
+  window.addEventListener('mousedown', resetTimer, false)
+  window.addEventListener('keypress', resetTimer, false)
+  window.addEventListener('DOMMouseScroll', resetTimer, false)
+  window.addEventListener('mousewheel', resetTimer, false)
+  window.addEventListener('touchmove', resetTimer, false)
+  window.addEventListener('MSPointerMove', resetTimer, false)
+
+  startTimer()
+}
+
+function startTimer() {
+  // wait 1 minute before calling goInactive
+  timeoutID = window.setTimeout(goInactive, 60000)
+}
+
+function resetTimer(e) {
+  window.clearTimeout(timeoutID)
+  goActive()
+}
+
+function goInactive() {
+  console.log('Going inactive due to 1 minute of no input')
+  window.clearTimeout(timeoutID)
+  setupStore().dispatch(changedFocus(false))
```

Comment:
store isn't in scope here, is it? I started passing it around to all these functions and handlers, but setupStore() is kind of a singleton retriever anyway, seemed harmless. what's a good way to do it?